src/Microsoft.SqlTools.ServiceLayer/Agent/JobUtilities.cs

```diff
@@ -0,0 +1,106 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Data;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.SqlServer.Management.Common;
+using Microsoft.SqlServer.Management.Sdk.Sfc;
+using Microsoft.SqlServer.Management.Smo.Agent;
+using Microsoft.SqlTools.ServiceLayer.Agent.Contracts;
+using SMO = Microsoft.SqlServer.Management.Smo;
+
+namespace Microsoft.SqlTools.ServiceLayer.Agent 
+{
+    public class JobUtilities
+    {
+        public const string urnJobName = "JobName";
```

Comment:
public const should typically be Pascal cased, so this should be UrnJobName.  I probably make all these consts follow this naming convention for consistency.